charts/firefly/README.md

```diff
@@ -119,30 +118,28 @@ swapping out the underlying blockchain and off-chain infrastructure easily.
+- Optional tokens connectors (ERC1155, ERC20, etc.)
 
 <p align="center">
   <img src="./../../img/helm_chart_deployment_architecture.jpg" width="75%" />
 </p>
 
 As depicted above, the chart only aims to provide a means for deploying the following components:
 
-
-| Component                                                                                    | Status           | Optional | Enabled by Default |
-|----------------------------------------------------------------------------------------------|------------------|----------|--------------------|
-| [FireFly Core]( https://github.com/hyperledger/firefly)                                      | GA               | ❌        | N/A                |
-| [FireFly Ethconnect]( https://github.com/hyperledger/firefly-ethconnect)                     | Deprecated       | ✅        | ❌                  |
-| [FireFly EVMConnect]( https://github.com/hyperledger/firefly-evmconnect)                     | ⍺lpha ⚠️         | ✅        | ❌                  |
-| [FireFly Fabconnect]( https://github.com/hyperledger/firefly-fabconnect)                     | Unimplemented 🙈 | N/A      | N/A                |
-| [FireFly DataExchange HTTPS]( https://github.com/hyperledger/firefly-dataexchange-https)     | GA               | ✅        | ✅                  |
-| [FireFly Tokens ERC1155]( https://github.com/hyperledger/firefly-tokens-erc1155)             | βeta             | ✅        | ❌                  |
-| [FireFly Tokens ERC20 / ERC721]( https://github.com/hyperledger/firefly-tokens-erc20-erc721) | βeta             | ✅        | ❌                  |
-
+| Component                                                                                   | Status           | Optional | Enabled by Default |
+| ------------------------------------------------------------------------------------------- | ---------------- | -------- | ------------------ |
+| [FireFly Core](https://github.com/hyperledger/firefly)                                      | GA               | ❌       | N/A                |
+| [FireFly Ethconnect](https://github.com/hyperledger/firefly-ethconnect)                     | Deprecated       | ✅       | ❌                 |
+| [FireFly EVMConnect](https://github.com/hyperledger/firefly-evmconnect)                     | ⍺lpha ⚠️         | ✅       | ❌                 |
+| [FireFly Fabconnect](https://github.com/hyperledger/firefly-fabconnect)                     | Unimplemented 🙈 | N/A      | N/A                |
+| [FireFly DataExchange HTTPS](https://github.com/hyperledger/firefly-dataexchange-https)     | GA               | ✅       | ✅                 |
+| [FireFly Tokens ERC1155](https://github.com/hyperledger/firefly-tokens-erc1155)             | βeta             | ✅       | ❌                 |
+| [FireFly Tokens ERC20 / ERC721](https://github.com/hyperledger/firefly-tokens-erc20-erc721) | βeta             | ✅       | ❌                 |
```

Comment:
This as well, not beta and it does get configured

Reply:
https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/213a84dfda8d8c097d37bd6ac7d3d2115f871048